design/trackers.go

```diff
@@ -0,0 +1,125 @@
+package design
+
+import (
+	d "github.com/goadesign/goa/design"
+	a "github.com/goadesign/goa/design/apidsl"
+)
+
+var trackerData = a.Type("TrackerData", func() {
```

Comment:
Hmm right, but the name `tracker` conflicts with existing `tracker` type. And that is being used widely in repositories. Old `tracker` type will be deprecated after tracker-CRUDL are compatible with jsonapi.